UniConnectApp/app/src/main/java/com/example/uniconnect/dto/Post.kt

```diff
@@ -1,6 +1,6 @@
 package com.example.uniconnect.dto
 
-data class Post(var title: String, var description: String) {
+data class Post(var postID: Int, var title: String, var description: String) {
```

Comment:
This will enable you to have posts that are titled the same but made at different times or places

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.